.travis.yml

```diff
@@ -5,10 +5,13 @@ env:
   matrix:
     - TOX_ENV=py27-django16
     - TOX_ENV=py27-django17
+    - TOX_ENV=py27-django18
     - TOX_ENV=py33-django16
     - TOX_ENV=py33-django17
+    - TOX_ENV=py33-django18
     - TOX_ENV=py34-django16
```

Comment:
No need to drop it now. If we need to implement something that requires a shim, then we can drop it then.